assets/queries/terraform/aws/s3_bucket_with_ignore_public_acl/query.rego

```diff
@@ -1,5 +1,7 @@
 package Cx
 
+import data.generic.common as common_lib
+
 CxPolicy[result] {
 	pubACL := input.document[i].resource.aws_s3_bucket_public_access_block[name]
 	pubACL.ignore_public_acls == true
```

Comment:
I think this query should give results if the opposite happens.. that is ignore_public_acls == false

https://docs.aws.amazon.com/AmazonS3/latest/userguide/access-control-block-public-access.html